src/commands/watch/filters.rs

```diff
@@ -185,3 +181,23 @@ fn standard_ignores() -> Vec<(String, Option<PathBuf>)> {
     .map(|pat| (pat.to_owned(), None))
     .collect()
 }
+
+#[derive(Debug)]
+struct CompositeFilterer {
+    filterers: Vec<Box<dyn watchexec::filter::Filterer>>,
+}
+
+impl watchexec::filter::Filterer for CompositeFilterer {
+    fn check_event(
+        &self,
+        event: &watchexec::event::Event,
+        priority: watchexec::event::Priority,
+    ) -> Result<bool, watchexec::error::RuntimeError> {
+        for f in &self.filterers {
+            if f.check_event(event, priority)? {
+                return Ok(true);
+            }
+        }
+        Ok(false)
+    }
+}
```

Comment:
Nit: A comment explaining that `any(event)` triggers this filter rather than `all(event)` might be nice.